joss/paper.md

```diff
@@ -1,51 +1,117 @@
+
 
+# Outline of `Eureka!`'s Stages
 
-# Statement of need
-Transit spectrsocopy is really hard!
+`Eureka!` is broken down into six stages which are as follows (also summarized in \autoref{fig:overview}):
 
+- Stage 1: An optional step that calibrates raw data (converts ramps to slopes for JWST observations). This step can be skipped if you'd rather use the `jwst` pipeline's Stage 1 outputs.
+- Stage 2: An optional step that further calibrates Stage 1 data (performs flat-fielding, unit conversion, etc. for JWST observations). This step can be skipped if you'd rather use the `jwst` pipeline's Stage 2 outputs.
+- Stage 3: Starts with the files produced by Stage 2 and performs background subtraction. This stage also reduces the data in order to convert 2D spectra into a time-series of 1D spectra or 2D photometric images into a 1D time-series.
+- Stage 4: Bins the 1D spectral time-series along the wavelength axis and generates light curves. Also removes 1D spectral drift/jitter and sigma clips outliers.
+- Stage 5: Fits the light curves with noise and astrophysical models using different optimization or sampling algorithms.
+- Stage 6: Creates a table and a figure summarizing the transmission and/or emission spectra from your Stage 5 fit(s).
 
-# Outline of the stages
+![An overview flowchart showing the processing done at each stage in `Eureka!`.\label{fig:overview}](figures/stages_flowchart.png){width=100%}
 
+# Statement of Need
+
+The calibration, reduction, and fitting of exoplanet time-series observations is a challenging problem with many tunable parameters across many stages, all of which may significantly impact the final results. Typically, the default calibration pipeline from astronomical observatories is insufficiently precise for exoplanet time-series observations as the pipeline is more optimized for other science use cases. As such, it is common practice to develop a custom data analysis pipeline that starts from the original, uncalibrated images. Historically, data analysis pipelines have often been proprietary, so each new user of an instrument or telescope has had to develop their own pipeline. Also, clearly specifying the analysis procedure used can be challenging, especially with proprietary code, which erodes reproducibility. `Eureka!` seeks to be a next-generation data analysis pipeline for next-generation observations from JWST with open-source and well-documented code for easier adoption; modular code for easier customization while maintaining a consistent framework; and easy-to-use but powerful inputs and outputs for increased automation, increased reproducibility, and more thorough intercomparisons. By also allowing for analyses of HST observations within the same framework, it will also be possible to combine new and old observations to develop a more complete understanding of individual targets or even entire populations.
```

Comment:
- "**many of which will** significantly impact the final results."
- "is insufficiently **tailored** for exoplanet time-series observations..."
- "within the same framework, **users will be able** to combine new and old observations..."